types/gestalt/index.d.ts

```diff
@@ -161,24 +183,20 @@ export interface BoxProps {
     mdPaddingY?: UnsignedUpTo12;
     lgPaddingY?: UnsignedUpTo12;
     position?: 'static' | 'absolute' | 'relative' | 'fixed';
-    ref?: React.Ref<'div'>;
```

Comment:
as per my other comment, why did you remove the `ref` prop from these components? My understanding is that it's needed to make `<Flyout />` show the popup on top of the right element - see https://gestalt.netlify.app/Flyout#basicExample